### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA announces additional inappropriate information when "Select Subscription" dialog is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1479]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign n" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to and select required subscription nad select start button.
6. In  "Create service principle Status" dialog ,tab to and select "OK" button.
7. Select Subscription dialog will open
8. Observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Select Subscription" dialog is opened 

**Jaws  behaviour:**
Select Subscriptions dialog
Subscription name Subscription ID 
Select Button
To activate press spacebar.

**NVDA Behaviour:**
Select Subscriptions  dialog
Select  button

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e Select Subscriptions dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![select subscriptions 2](https://user-images.githubusercontent.com/22956130/36105160-f4011a4e-1039-11e8-9f0c-c937c3604a7f.jpg)
![select subscription-nvda](https://user-images.githubusercontent.com/22956130/36105166-f79c939a-1039-11e8-9d5c-27aa8671059b.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749912
